dns-records/jitaccess.ops.phac.alpha.canada.ca.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: dns.cnrm.cloud.google.com/v1beta1
+kind: DNSRecordSet
+metadata:
+  name: jitaccess-ops-phac-alpha-canada-ca-ns
+  namespace: alpha-dns
+  labels:
+    ownerName: "Eric Malenfant"
+    ownerContact: "eric.richard-malenfant@phac.aspc.gc.ca"
+    gitHubRepo: "JIT-IaC"

```

Comment:
could you change that to a URL, even if it's private. I will ask that @sleepycat confirm that's fine.